src/lib/sql/sql_pipeline_statement.cpp

```diff
@@ -159,7 +156,9 @@ const std::shared_ptr<SQLQueryPlan>& SQLPipelineStatement::get_query_plan() {
     Assert(_prepared_statements, "Cannot execute statement without prepared statement cache.");
     const auto plan = _prepared_statements->try_get(execute_statement->name);
 
-    Assert(plan, "Requested prepared statement does not exist!");
+    if (!plan) {
```

Comment:
These one-line `if () throw` statements should probably be turned into `AssertInput`